app/components/conferences/display/EventShow.js

```diff
@@ -59,6 +59,7 @@ export const EventShow = ({ event, error, speaker, prsession }) => {
               <div className={styles.event_logo}>
                 <Image src={event.data.attributes["logo-url"]} width={100} />
               </div>
+              <div style={{textAlign: 'center'}}><h6> Organizer </h6></div>
```

Comment:
Please use `className` for styling since if we use inline styling, the customization of the components' style will be affected.

Because for any style changes needed, the user would have to do modifications in the component (`EventBasicDetail.js`)  in case of inline styling; However, for separate `className` in a `.css` file, they can modify a single CSS class property in the `style.css` which is pretty convenient.

Thank you!

Reply:
got it! please check the changes now...
Thank you!